src/main/java/core/basesyntax/FileReaderImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader{
```

Comment:
There is an unnecessary empty line at the beginning of the class implementation. Please remove it.